File: Lista de presença/Exe 1.c
#include <stdio.h>

int main() {
    char nome[100];
    float salarioAtual, novoSalario;
    int percentualAumento;

    printf("Digite seu nome: ");
    fgets(nome, sizeof(nome), stdin);

    printf("Digite o seu salário atual (R$): ");
    scanf("%f", &salarioAtual);

    if (salarioAtual <= 400.00) {
        percentualAumento = 15;
    } else if (salarioAtual <= 700.00) {
        percentualAumento = 12;
    } else if (salarioAtual <= 1000.00) {
        percentualAumento = 10;
    } else if (salarioAtual <= 1800.00) {
        percentualAumento = 7;
    } else if (salarioAtual <= 2500.00) {
        percentualAumento = 4;
    } else {
        percentualAumento = 0;
    }

    novoSalario = salarioAtual + (salarioAtual * percentualAumento / 100.0);

    printf("Informações do Funcionário:\n");
    printf("Nome: %s", nome);
    printf("Percentual de aumento: %d%%\n", percentualAumento);
    printf("Salário atual: R$ %.2f\n", salarioAtual);
    printf("Novo salário: R$ %.2f\n", novoSalario);

    return 0;
}

File: Lista de presença/Exe 1.py
nome = input("Digite seu nome: ")
SalarioAtual = float(input("Digite o seu salário atual (R$): "))

NovoSalario = SalarioAtual
PercentualAumento = 0


if SalarioAtual <= 400.00:
    PercentualAumento = 15
elif SalarioAtual <= 700.00:
    PercentualAumento = 12
elif SalarioAtual <= 1000.00:
    PercentualAumento = 10
elif SalarioAtual <= 1800.00:
    PercentualAumento = 7
elif SalarioAtual <= 2500.00:
    PercentualAumento = 4
else:
    PercentualAumento = 0

NovoSalario = SalarioAtual + (SalarioAtual * PercentualAumento / 100)

print("Informações do Funcionário:")
print("Nome:", nome)
print("Percentual de aumento:", PercentualAumento, "%")
print("Salário atual: R$", SalarioAtual)
print(f"Novo salário: R$ {SalarioAtual:.2f}")